### DISCUSSION TITLE
Upcoming development of v4

### DISCUSSION THREAD

**Discussion by igor-baiborodine** [Lednerb/bilberry-hugo-theme#523] [General]
I'm happy to announce the upcoming development of the `v4` version of the Bilberry Hugo theme. It will contain the following upgrades, improvements, and enhancements:
1. Migrate from highlight.js to Hugo's built-in code highlighting; see https://github.com/Lednerb/bilberry-hugo-theme/issues/359 and https://github.com/Lednerb/bilberry-hugo-theme/discussions/362.
2. Add from the bottom to the top arrow/button; see https://github.com/Lednerb/bilberry-hugo-theme/discussions/487
3. Migrate to the Hugo way of adding menu items in the top navigation; see https://github.com/Lednerb/bilberry-hugo-theme/discussions/437.
4. Upgrade Font Awesome icons to `v6`; see https://github.com/Lednerb/bilberry-hugo-theme/discussions/358.
5. Migrate from npm to Hugo pipelines for SCSS processing; see https://github.com/Lednerb/bilberry-hugo-theme/discussions/339.

@Lednerb, let me know what you think. Did I miss anything?

**Comment by Lednerb** [comment]
Hi @igor-baiborodine thanks for your ongoing efforts in this project!

This is a great roadmap and at the moment I do not have any further additions.

**Reply by igor-baiborodine** [reply]
I created the [Development of v4 version project](https://github.com/Lednerb/bilberry-hugo-theme/projects/2) to track the progress.
